kiwi/bootloader/config/bootloader_spec_base.py

```diff
@@ -0,0 +1,215 @@
+# Copyright (c) 2022 Marcus Schäfer
+#
+# This file is part of kiwi.
+#
+# kiwi is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kiwi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kiwi.  If not, see <http://www.gnu.org/licenses/>
+#
+from typing import (
+    Dict, NamedTuple
+)
+
+# project
+from kiwi.bootloader.config.base import BootLoaderConfigBase
+
+target_type = NamedTuple(
+    'target_type', [
+        ('disk', str),
+        ('install', str),
+        ('live', str)
+    ]
+)
+
+
+class BootLoaderSpecBase(BootLoaderConfigBase):
+    """
+    **Base class for Boot Loader Specification**
+
+    This base class follows the bootloader specifications as
+    described in https://systemd.io/BOOT_LOADER_SPECIFICATION
```

Comment:
You might want to link to https://uapi-group.org/specifications/specs/boot_loader_specification/ instead which is the new location for the boot loader specification

Reply:
done